### PR TITLE
Task 52011 booking history

### PR DESCRIPTION
**Pull request by luctq-0138**
## Related Tickets
- [#TicketID](https://edu-redmine.sun-asterisk.vn/issues/52011)

## WHAT (optional)
- Change number items `completed/total` in admin page.

## HOW
- I edit js file, inject not_vary_normal items in calculate function.

## WHY (optional)
- Because in previous version - number just depends on `normal` items. But in new version, we have `state` and `confirm_state` depends on both `normal` + `not_normal` items.

### CODE REVIEW

**Review by VuTan115 — APPROVED** [review]

**Comment by luctq-0138** [comment]
ready

File: app/views/bookings/_booking.html.erb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<tr>
+  <% if booking.status != 3 %>
```
Comment (ngocvt-0484) [inline app/views/bookings/_booking.html.erb]: check kiểu enum e ơi
3 là cái gì?

File: app/views/bookings/_booking.html.erb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    <td><%= booking.booking_date %></td>
+    <td><%= booking.check_in %></td>
+    <td><%= booking.check_out %></td>
+    <td>
+      <% if booking.status == 0%>
```
Comment (ngocvt-0484) [inline app/views/bookings/_booking.html.erb]: same

**Comment by luctq-0138** [comment]
ready

**Pull request merged**